pkg/resourcemanager/keyvaults/keyvault.go

```diff
@@ -250,7 +250,7 @@ func InstantiateVault(ctx context.Context, vaultName string, containsUpdate bool
 }
 
 // CreateVault creates a new key vault
-func (k *azureKeyVaultManager) CreateVault(ctx context.Context, instance *v1alpha1.KeyVault, sku azurev1alpha1.KeyVaultSku, tags map[string]*string) (keyvault.Vault, error) {
+func (k *azureKeyVaultManager) CreateVault(ctx context.Context, instance *v1alpha1.KeyVault, sku azurev1alpha1.KeyVaultSku, tags map[string]*string, exists bool) (keyvault.Vault, error) {
```

Comment:
What does the `exists` parameter mean? perhaps a longer parameter name would be clearer?

Reply:
`exists` signifies that the Vault doesn't need to be created so it probably needs its policies updated. Happy to make the var name more detailed